docs/examples.md

```diff
@@ -0,0 +1,363 @@
+                                   :props {:position "Developer"}})
+~~~
+
+### Find relationships in Neo4j
+
+To fetch relationships from the Neo4j database, please refer to the section [Get Graph](#get-graph)
+
+### Update and Delete Relationship
+
+See the section [Update and Delete Entity](#update-and-delete-neo4j-entity)
+
+
+## Update and Delete Neo4j Entity
+
+In this section you can find examples on how to update specific parts a Node or Relationship and how to delete them.
+
+### Update Node (Labels)
+
+To change the labels of a node we have added two convenience functions.
+Both the `add-labels!` and `remove-labels!` functions takes a lookup representation as second argument.
+To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+#### Add labels
+
+~~~clojure
+(client/add-labels! conn {:id 45} [:person :salesman])
+~~~
+
+#### Remove labels
+
+~~~clojure
+(client/remove-labels! conn {:labels [:person] :props {:first-name "Neo"}} [:person :salesman])
+~~~
+
+### Update Node or Relationship (properties)
+
+To change the properties of a node or relationship we have added two convenience functions.
+Both the `update-properties!` and `replace-properties!` functions takes a lookup representation as second argument.
+
+To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+#### Update properties
+
+This function will update an existing properties map based on the following rules.
+
+⋅⋅* Keys existing only in the given property map is added to the object
+..* Keys existing only in the property map on the found object is kept as is
+..* Keys existing in both property maps are updated with values from the given property map
+
+~~~clojure
+(client/update-props! conn {:labels [:person] :first-name "Thomas"} {:first-name "Neo"})
+~~~
+
+#### Replace properties
+
+This function will replace an existing property map with the given one.
+
+~~~clojure
+(client/replace-props! conn {:labels [:person] :first-name "Thomas"} {:last-name-only "Anderson"})
+~~~
+
+### Delete Node or Relationship
+
+The `delete!` function takes a lookup representation as second argument and deletes all matches.
+
+Notice: You need to ensure you have deleted all relatiohsips to a node before you can delete the node.
+
+To learn more about the Clojure representation of a lookup entry please see our [representations](representations.md) page
+
+~~~clojure
+(client/delete! conn {:labels [:person] :first-name "Neo"})
+~~~
+
+
+## Graph CRUD
+
+This section shows how to do basic CRUD operations on a whole graph through Neo4clj convenience functions.
+
+### Create Graph
+
+To make it easier to create nodes and relationships, we have made a function to do it in one single call.
+
+The `lookups` key allows you to lookup and refer existing nodes in the Neo4j database. The entries are lookup representations.
```

Comment:
Here (I assume) the word "entries" refers to an item in the collection (vector)? I is a bit confusing that we are using the word `Entity` for a common term covering both nodes and relationships but now it is being used to cover something else. Or maybe I'm wrong?

Reply:
This have been removed in the rewrite of the descriptions